jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -253,17 +253,41 @@ bool ci_lazy_methodst::handle_virtual_methods_with_no_callees(
     if(!candidate_target_methods.empty())
       continue;
 
+    const java_method_typet &java_method_type =
+      to_java_method_type(virtual_function_call.type());
```

Comment:
:pick: I prefer having definitions closer to where there are used (i.e. line 268)